app/javascript/src/components/menus/page_menu.js

```diff
@@ -111,23 +111,34 @@ class PageMenu extends ActivatedMenu {
     var $link = element.find("> a");
     new DialogApiRequest($link.attr("href"), {
       activator: $link,
-      closeOnClickSelector: ".govuk-button",
-      build: function(dialog) {
+      closeOnClickSelector: "button.govuk-button",
+      onLoad: function(dialog) {
         // Find and correct (make work!) any method:delete links
         dialog.$node.find("[data-method=delete]").on("click", function(e) {
           e.preventDefault();
-          post(this.href, { _method: "delete" });
+          console.log(this.dataset.url);
```

Comment:
Should this be commented out or just removed?

Reply:
dammit! I did a grep for all console.logs! how did I miss one??

Reply:
Oh, it is gone. this block is listed as outdated...